### PR TITLE
Grammar fixes for landing page + blog posts

### PR DESCRIPTION
**Pull request by proudmuslim-dev**

### CODE REVIEW

File: _posts/2021-10-1-updates.md [hunk 1]
```diff
@@ -26,24 +26,24 @@ We track the progress on this front on Github:
...
 
 ## Bounties
 
-If you wish to work in exchange for XMR take a look at our bounty program. It's already up and running!
+If you wish to work in exchange for XMR, take a look at our bounty program. It's already up and running!
 
 The issues avialable for a bounty in XMR are tracked on a dedicated Kanban board on Github: https://github.com/orgs/haveno-dex/projects/2
 
-Would you like to contribute to Haveno in exchange for XMR but there is nothing that suit you yet? Just contact us and let us know what would you like to work on, if we think it worth it, we will gladly add a bounty for the job. See the [conditions for receiving a bounty](https://github.com/haveno-dex/haveno/blob/master/docs/bounties.md).
+Would you like to contribute to Haveno in exchange for XMR but can't because there is nothing that suits you yet? Just contact us and let us know what would you like to work on- if we think it's worth it, we will gladly add a bounty for the job. See the [conditions for receiving a bounty](https://github.com/haveno-dex/haveno/blob/master/docs/bounties.md) for more information.
```
Comment (erciccione) [inline _posts/2021-10-1-updates.md:37]: I'm not a big fan of using `-` after words (in this case `work on-`). I would prefer to use a period or a comma.

File: _posts/2021-07-7-bisq-vulnerability.md [hunk 2]
```diff
@@ -4,18 +4,18 @@ title:  "Vulnerability in Bisq found and disclosed"
...
+All the data could have been harvested at no cost at all for the attacker. We don't know how long Bisq has had this vulnerability for and if it was exploited. We know that it was probably not exploited at a large scale, or Bisq's support team would have noticed a spike in support tickets.
 
-We reported the vulnerability to Bisq and helped them to patch it. They then released a patched version of their software, which fixed the problem (v1.7.0).
+We reported the vulnerability to Bisq and helped them patch it. They then released a patched version of their software, which fixed the problem (v1.7.0).
 
 The disclosure on Haveno's Twitter: [https://twitter.com/HavenoDEX/status/1412782658887766016](twitter.com/HavenoDEX/status/1412782658887766016).
 
-The disclosure on Bisq's Twitter: [https://twitter.com/bisq_network/status/1412536933910147072](twitter.com/bisq_network/status/1412536933910147072).
\ No newline at end of file
+The disclosure on Bisq's Twitter: [https://twitter.com/bisq_network/status/1412536933910147072](twitter.com/bisq_network/status/1412536933910147072).
```
Comment (erciccione) [inline _posts/2021-07-7-bisq-vulnerability.md:21]: Noticing now that this link is wrong. Please change it to

```suggestion
The disclosure on Bisq's Twitter: [twitter.com/bisq_network/status/1412536933910147072](https://twitter.com/bisq_network/status/1412536933910147072).
```

File: _posts/2022-02-2-haveno-structure.md [hunk 3]
```diff
@@ -46,20 +46,20 @@ Arbitrators are a fundamental part of Haveno and will resolve trade disputes. Th
...
+The important thing for us is to keep Haveno **decentralized and community focused**. We will always be looking to improve these aspects- that's why we don't exclude different governance systems in the future. For example, it could be interesting to plug Haveno into an external DAO for governance. It could be even more interesting to build this DAO on Monero! In the future we could have colored coins on the Monero blockchain or other mechanisms to empower the community. There are infinite possibilities. The point is that with the constant stream of funds, Haveno will provide: we will open the doors to a lot of possibilities that went unexplored until now because of the absence of a reliable stream of funds.
```
Comment (erciccione) [inline _posts/2022-02-2-haveno-structure.md:57]: See my comment above about using `-` after words.

**Review by erciccione — COMMENTED** [review]
Thanks. This is very appreciated :) I left some comments.

File: _posts/2021-07-7-bisq-vulnerability.md (hunk 2 above)
Comment (proudmuslim-dev) [reply _posts/2021-07-7-bisq-vulnerability.md:21]: Alright, will do

File: _posts/2021-10-1-updates.md (hunk 1 above)
Comment (proudmuslim-dev) [reply _posts/2021-10-1-updates.md:37]: Alright

**Review by erciccione — APPROVED** [review]
Thanks :)

**Comment by proudmuslim-dev** [comment]
No problem! :)

**Pull request merged**